lib/fastlane/actions/cert.rb

```diff
@@ -0,0 +1,33 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      CERT_FILE_PATH = :CERT_FILE_PATH
+      CERT_CERTIFICATE_ID = :CERT_CERTIFICATE_ID
+    end
+
+    class CertAction
+      def self.run(params)
+        require 'cert'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.